app/electron/main.ts

```diff
@@ -315,7 +315,7 @@ function startElecron() {
           owner: 'kinvolk',
           repo: 'headlamp',
         });
-        if (response.data.name !== appVersion) {
+        if (response.data.name >= appVersion && !response.data.name.startsWith('headlamp-helm')) {
```

Comment:
This means the update_available will only work until a chart is published. The logic for getting the latest release needs to be to get the latest N releases (or all, if there's no such API) and get the latest in the list that's starting with `v*`.

Reply:
fixed